docs/guides/deployment/push-work-pools.md

```diff
@@ -19,12 +19,15 @@ Push [work pools](/concepts/work-pools/#work-pool-overview) are a special type o
 
 In this guide you will:
 
-- Create a push work pool that sends work to Google Cloud Run, Amazon Elastic Container Service (AWS ECS) or Azure Container Instances (ACI)
+- Create a push work pool that sends work to Google Cloud Run, Amazon Elastic Container Service (AWS ECS), or Azure Container Instances (ACI)
 - Deploy a flow to that work pool
-- Execute our flow without having to run a worker or agent process to poll for flow runs
+- Execute the flow without having to run a worker or agent process to poll for flow runs
 
 ## Setup
 
+!!! note "Automatic infrastructure provisioning"
+    You can automatically provision infrastructure and create your push work pool using the `prefect work-pool create` CLI command with the `--provision-infra` flag. See the [Automatic Infrastructure Provisioning guide](/guides/deployment/auto-infra/) for more details.
```

Comment:
I think we could simplify this page a lot if we had `--provision-infra` as the default way to create push work pools. Could we move the info from the new page here instead?

Reply:
> I think we could simplify this page a lot if we had --provision-infra as the default way to create push work pools. 

That's an interesting idea. I was assuming we'd want a single page to discuss all the infra provisioning options as we add more of them. That might not be a good assumption.

Reply:
We are scoping infrastructure provisioning to just push pools for now, so I think we can keep it all in the push pools page.

Reply:
Ah, good to know. 
I pushed everything into the push pools page. I left the manual examples lower on the page. With tabs I think the amount of content stays manageable.